cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/util/AwsMethodExecutor.java

```diff
@@ -21,13 +21,15 @@
     public <T> T execute(Supplier<T> awsMethod, T def) {
         T ret;
         try {
+            LOGGER.debug("Fetch data from AWS");
```

Comment:
don't we have some flag for AWS debug log?

Reply:
I don't know, I didn't see any kind of this log. But If you can show me, I will remove this